packages/eslint-plugin/tests/rules/prefer-readonly.test.ts

```diff
@@ -717,6 +717,15 @@ class Foo {
         }
       }
     `,
+    `
+      class TestAccessors {
+        private static accessor staticAcc = 1;
+        static accessor #staticAcc = 1;
+
+        private accessor acc = 3;
+        accessor #acc = 3;
+      }
+    `,
```

Comment:
[Testing] Per https://typescript-eslint.io/contributing/local-development#granular-unit-tests we generally prefer unit tests be more granular: each of these fields would ideally be their own test case. Could you please split these up?

I do see that the rest of this file doesn't exactly adhere to the granularity thing 🥲 - there are a lot of files that don't, unfortunately.

Reply:
Done.